stdlib/public/Concurrency/TaskGroup+addTask.swift.gyb

```diff
@@ -0,0 +1,408 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Swift
+
+// ==== addTask / addTaskUnlessCancelled ---------------------------------------
```

Comment:
This gyb replaces a few thousand lines (and like 20 definitions) of addTask implementations. We still have ad hoc ones though, and I can't get the gyb result to be included in the build so there's workarounds here still